README.md

```diff
@@ -374,26 +348,26 @@ indicates the check was completely sure of the result.
-[CII-Best-Practices](docs/checks.md#cii-best-practices)          | Does the project have a [CII Best Practices Badge](https://bestpractices.coreinfrastructure.org/en)?
-[Code-Review](docs/checks.md#code-review)                 | Does the project require code review before code is merged?
-[Contributors](docs/checks.md#contributors)                | Does the project have contributors from at least two different organizations?
-[Dangerous-Workflow](docs/checks.md#dangerous-workflow)          | Does the project avoid dangerous coding patterns in GitHub Action workflows?
-[Dependency-Update-Tool](docs/checks.md#dependency-update-tool)      | Does the project use tools to help update its dependencies?
-[Fuzzing](docs/checks.md#fuzzing)                     | Does the project use fuzzing tools, e.g. [OSS-Fuzz](https://github.com/google/oss-fuzz)?
-[License](docs/checks.md#license)                     | Does the project declare a license?
-[Maintained](docs/checks.md#maintained)                  | Is the project maintained?
-[Pinned-Dependencies](docs/checks.md#pinned-dependencies)         | Does the project declare and pin [dependencies](https://docs.github.com/en/free-pro-team@latest/github/visualizing-repository-data-with-graphs/about-the-dependency-graph#supported-package-ecosystems)?
-[Packaging](docs/checks.md#packaging)                   | Does the project build and publish official packages from CI/CD, e.g. [GitHub Publishing](https://docs.github.com/en/free-pro-team@latest/actions/guides/about-packaging-with-github-actions#workflows-for-publishing-packages) ?
-[SAST](docs/checks.md#sast)                        | Does the project use static code analysis tools, e.g. [CodeQL](https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/enabling-code-scanning-for-a-repository#enabling-code-scanning-using-actions), [LGTM](https://lgtm.com), [SonarCloud](https://sonarcloud.io)?
-[Security-Policy](docs/checks.md#security-policy)             | Does the project contain a [security policy](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository)?
-[Signed-Releases](docs/checks.md#signed-releases)             | Does the project cryptographically [sign releases](https://wiki.debian.org/Creating%20signed%20GitHub%20releases)?
-[Token-Permissions](docs/checks.md#token-permissions)           | Does the project declare GitHub workflow tokens as [read only](https://docs.github.com/en/actions/reference/authentication-in-a-workflow)?
-[Vulnerabilities](docs/checks.md#vulnerabilities)             | Does the project have unfixed vulnerabilities? Uses the [OSV service](https://osv.dev).
+Name                        | Description | Risk Level
```

Comment:
IMO this table could be sorted by `High`, `Medium` and `Low`

Reply:
Good idea, and then alphabetically within each risk level.